centreon/www/install/php/Update-23.10.2.php

```diff
@@ -49,9 +49,24 @@
 };
 
 try {
+    $dropColumnVersionFromDashboardWidgetsTable($pearDB);
+
```

Comment:
```suggestion
```
This has been moved to 22.10.3 thats why your e2e tests failed

"Value of type null is not callable"